packages/app/src/ServicesDashboardPage.tsx

```diff
@@ -1609,6 +1656,15 @@ function ServicesDashboardPage() {
           </Tabs.Panel>
         </Tabs>
       )}
+      <DashboardFiltersModal
```

Comment:
design: Maybe it's just me.. but my brain spent a couple extra seconds looking for the "Done" button. I accidentally hti add new filter after once. Maybe it should show the cards, then "Add new filter left aligned below, then done on the right hand side? 

<img width="489" height="273" alt="Image" src="https://github.com/user-attachments/assets/76321735-da9a-46c7-bd08-1e17e6487b52" />